frontend/src/v5/store/containers/containers.sagas.ts

```diff
@@ -66,8 +69,12 @@ export function* fetchContainers({ teamspace, projectId }: FetchContainersAction
 	try {
 		const { containers }: FetchContainersResponse = yield API.Containers.fetchContainers({ teamspace, projectId });
 		const containersWithoutStats = prepareContainersData(containers);
+		const storedContainers = yield select(selectContainers);
```

Comment:
If you have no containers in a project then the list remains pending forever. The same is true for no federations in fetchFederations